### PR TITLE
working version of AWS connector with websocket and dynamoDB

### PR DESCRIPTION
**Pull request by dhruvsoni1802**

### CODE REVIEW

File: getSignedURL/app.ts [hunk 1]
```diff
@@ -79,6 +79,52 @@ const getUploadURL = async function (event) {
...
+    FunctionName: process.env.SendMessage as string,
+    InvocationType: "RequestResponse",
+    Payload: JSON.stringify(event),
+  };
+
+  try {
+    const data:any = await lambda.invoke(params).promise();
+    const result = JSON.parse(data.Payload);
+    return result;
+  } catch (error) {
```
Comment (jchris) [inline getSignedURL/app.ts]: you can remove this try catch

File: onconnect/app.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+  SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+"use strict";
+import AWS from "aws-sdk";
+import { DynamoDBClient, QueryCommand } from "@aws-sdk/client-dynamodb";
+import {
+  DynamoDBDocumentClient,
+  ScanCommand,
+  PutCommand,
```
Comment (jchris) [inline onconnect/app.ts:22]: only uses Put?

File: ondisconnect/app.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+*/
+"use strict";
+import AWS from "aws-sdk";
+import { DynamoDBClient, QueryCommand } from "@aws-sdk/client-dynamodb";
+import {
+  DynamoDBDocumentClient,
+  ScanCommand,
+  PutCommand,
+  GetCommand,
+  DeleteCommand,
```
Comment (jchris) [inline ondisconnect/app.ts:22]: only delete?

File: sendmessage/app.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+  try {
+    connectionData = await ddb
+      .scan({
+        TableName: TABLE_NAME,
+        ProjectionExpression: "connectionId",
+        FilterExpression: "databasename = :dbName",
+        ExpressionAttributeValues: {
+          ":dbName": event.databasename,
+        },
```
Comment (jchris) [inline sendmessage/app.ts:24]: this will look good in the blog

**Review by jchris — CHANGES_REQUESTED** [review]
this is rad. I put a few changes then we can merge

File: onconnect/app.ts (hunk 2 above)
Comment (dhruvsoni1802) [reply onconnect/app.ts:22]: Yes onconnect is there just to store the live connection ids of clients into subscribers table

File: ondisconnect/app.ts (hunk 3 above)
Comment (dhruvsoni1802) [reply ondisconnect/app.ts:22]: The opposite of onconnect function

File: getSignedURL/app.ts (hunk 1 above)
Comment (dhruvsoni1802) [reply getSignedURL/app.ts]: Okk

**Pull request merged**